### DISCUSSION TITLE
Welcome to PyThaiNLP Discussions!

### DISCUSSION THREAD

**Discussion by wannaphong** [PyThaiNLP/pythainlp#496] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 สวัสดี!
พวกเราใช้ระบบ Discussions เป็นสถานที่ไว้สำหรับเชื่อมต่อสมาชิกคนอื่น  ๆ กับชุมชน PyThaiNLP ของเรา พวกเราหวังว่าคุณ:
  * ถามคำถามที่คุณต้องการคำตอบ
  * แบ่งปันไอเดีย
  *  มีส่วนร่วมกับสมาชิกชุมชนคนอื่น ๆ
   * ยินดีต้อนรับผู้อื่นและเปิดใจกว้าง โปรดจำไว้ว่านี่คือชุมชนที่เราสร้างด้วยกัน💪.

ในการเริ่มต้นให้แสดงความคิดเห็นด้านล่างพร้อมกับการแนะนำตัวเองและบอกเราเกี่ยวกับสิ่งที่คุณทำกับชุมชนนี้

คุณสามารถใช้ได้ทั้งภาษาไทยและภาษาอังกฤษ เลือกภาษาที่คุณถนัด 😄

**Comment by wannaphong** [comment]
Hello! My name is Mr.Wannaphong Phatthiyaphaibun. I founded PyThaiNLP.

**Comment by wannaphong** [comment]
ใช้บัญชี GitHub ในการตอบและตั้งกระทู้

**Comment by BananaCodeSucker** [comment]
ขอสอบถามหน่อยนะครับ
พอดีผมต้องการจะนำ source code pythainlp-2.2.4
ไปพัฒนาต่อนะครับ
อยากทราบว่า
จะต้องใช้ editor ตัวไหนในการเปิด project นะครับ
ช่วยแนะนำหน่อยขอบคุณครับ
ขอบคุณครับ

**Reply by wannaphong** [reply]
ใช้ [VS code](https://code.visualstudio.com/) ได้เลยครับ โดยเลือก open folder เลือกโฟลเดอร์โค้ด pythainlp ครับ

**Reply by BananaCodeSucker** [reply]
ขอบคุณมากครับ
Open project ได้แล้วครับ

**Comment by Ar3ee3ar** [comment]
ขอสอบถามว่าสามารถใช้ library PyThaiNLP ในการทำโปรเจคจบได้ไหมคะ แล้วมีการเพิ่มคำศัพท์ด้วยค่ะ

**Comment by wannaphong** [comment]
ได้นะครับเมื่อ 13 ก.พ. 2021 18:00 น. Arzeezar Lording <notifications@github.com> เขียนว่า

ขอสอบถามว่าสามารถใช้ library PyThaiNLP ในการทำโปรเจคจบได้ไหมคะ

**Reply by Ar3ee3ar** [reply]
ขอบคุณค่ะ
